mtprotoproxy.py

```diff
@@ -119,10 +120,22 @@ def decrypt(self, data):
 KEY_LEN = 32
 IV_LEN = 16
 HANDSHAKE_LEN = 64
-MAGIC_VAL_POS = 56
+PROTO_TAG_POS = 56
 DC_IDX_POS = 60
 
-MAGIC_VAL_TO_CHECK = b'\xef\xef\xef\xef'
+PROTO_TAG_ABRIDGED = b'\xef\xef\xef\xef'
+PROTO_TAG_INTERMEDIATE = b'\xee\xee\xee\xee'
+
+class RpcFlags(enum.Flag):
+    NONE                  =        0x0
+    NOT_ENCRYPTED         =        0x2
+    HAS_AD_TAG            =        0x8
+    MAGIC                 =     0x1000
+    EXTMODE2              =    0x20000
+    DROPPED               = 0x10000000
```

Comment:
This doesn't seem to affect anything except that the reference implementation closes the connection if it sees a dropped flag, which this one does already